tests/psa_helpers.function

```diff
@@ -32,6 +32,36 @@
  */
 #define PSA_ASSERT( expr ) TEST_EQUAL( ( expr ), PSA_SUCCESS )
 
+static void test_helper_psa_done( int line, const char *file )
+{
+    mbedtls_psa_stats_t stats;
+    const char *msg = NULL;
+
+    mbedtls_psa_get_stats( &stats );
+
+    if( stats.volatile_slots != 0 )
+        msg = "A volatile slot has not been closed properly.";
```

Comment:
Such checks cover each other in case of multiple failures at once.

Reply:
Indeed. The message has to be a string literal because the test infrastructure stores a pointer to it which it prints and never frees. Within this constraint, do you have a better suggestion? This is only a matter of reporting failures, and most tests will only have one failure to report in practice.

Reply:
I would suggest concatenating the errors, but since this is for test purpose only, it's not a blocker.

Reply:
Unfortunately, we can't concatenate the errors unless we provide a string literal for each possible combination of issues. It would be nice, yes, but I don't think it's worth it to make major changes to test failure reporting just for that.